x/vesting/keeper/integration_test.go

```diff
@@ -0,0 +1,515 @@
+	}
+	app.FeeMarketKeeper.SetBaseFee(suite.ctx, sdk.ZeroInt().BigInt())
+
+	// Set Validator
+	valAddr := sdk.ValAddress(address.Bytes())
+	validator, err := stakingtypes.NewValidator(valAddr, privCons.PubKey(), stakingtypes.Description{})
+	if err != nil {
+		return err
+	}
+	validator = stakingkeeper.TestingUpdateValidator(&app.StakingKeeper, suite.ctx, validator, true)
+	err = app.StakingKeeper.Hooks().AfterValidatorCreated(suite.ctx, validator.GetOperator())
+	if err != nil {
+		return err
+	}
+	err = app.StakingKeeper.SetValidatorByConsAddr(suite.ctx, validator)
+	if err != nil {
+		return err
+	}
+
+	suite.querier = keeper.Querier{Keeper: app.VestingKeeper}
+	suite.msgServer = keeper.NewMsgServerImpl(app.VestingKeeper)
+	suite.stkQuerier = stakingkeeper.Querier{Keeper: &app.StakingKeeper}
+
+	validators := app.StakingKeeper.GetValidators(suite.ctx, 2)
+	// Set a bonded validator that takes part in consensus
+	if validators[0].Status == stakingtypes.Bonded {
+		suite.validator = validators[0]
+	} else {
+		suite.validator = validators[1]
+	}
+	return nil
+}
+
+var _ = Describe("Monthly Vesting Account", Ordered, func() {
+	const (
+		cliffDays = 30
+		months    = 3
+	)
+
+	var (
+		s *VestingTestSuite
+
+		vaPrivKey cryptotypes.PrivKey // private key of vesting account
+		va        sdk.AccAddress      // vesting account
+		funder    sdk.AccAddress      // funder account who initiates monthly vesting
+		user      sdk.AccAddress
+
+		initialVesting sdk.Coins // initial vesting coins
+		extraCoins     sdk.Coins // additional funds to pay gas fees for tx
+		mva            *types.MonthlyVestingAccount
+
+		unvested sdk.Coins
+		vested   sdk.Coins
+	)
+
+	BeforeEach(func() {
+		var err error
+
+		s = new(VestingTestSuite)
+		err = s.SetupTest()
+		Expect(err).To(BeNil())
+
+		now := s.ctx.BlockTime()
+
+		var from common.Address
+		from, vaPrivKey = tests.RandomEthAddressWithPrivateKey()
+		va = sdk.AccAddress(from.Bytes())
+		user = tests.RandomAccAddress()
+		funder = tests.RandomAccAddress()
+
+		amount := math.NewInt(1e17).Mul(math.NewInt(months))
+		initialVesting = sdk.NewCoins(sdk.NewCoin(utils.BaseDenom, amount))
+
+		// Fund coins to funder to create monthly vesting
+		err = testutil.FundAccount(s.ctx, s.app.BankKeeper, funder, initialVesting)
+		Expect(err).To(BeNil())
+		err = s.Commit()
+
+		// Create monthly vesting
+		resp, err := s.msgServer.HandleCreateMonthlyVestingAccount(s.goCtx, &types.MsgCreateMonthlyVestingAccount{
+			FromAddress: funder.String(),
+			ToAddress:   va.String(),
+			CliffDays:   cliffDays, // 30 days
+			Months:      months,    // 3 months
+			Amount:      initialVesting,
+		})
+		Expect(resp).To(Equal(&types.MsgCreateMonthlyVestingAccountResponse{}))
+		err = s.Commit()
+		Expect(err).To(BeNil())
+
+		// Query monthly vesting account
+		mva, err = s.querier.GetMonthlyVestingAccount(s.ctx, va)
+		Expect(err).To(BeNil())
+		Expect(mva).ToNot(BeNil())
+
+		//baseAcc := authtypes.NewBaseAccountWithAddress(va)
```

Comment:
Please remove commented code

Reply:
The commented code is basically same with creating vesting account and querying vesting account.
It was replaced by `s.msgServer.HandleCreateMonthlyVestingAccount` and `s.querier.GetMonthlyVestingAccount`.